src/components/NegotiationList.vue

```diff
@@ -229,17 +237,66 @@
             })
           "
         />
+
+<!-- // To do add table for different view -->
```

Comment:
please remove commented code or move it to a new PR.

Reply:
removed